<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tableau to Power BI Migrator</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>451</width>
      <height>601</height>
     </rect>
    </property>
    <property name="title">
     <string>Control Panel</string>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>431</width>
       <height>231</height>
      </rect>
     </property>
     <property name="title">
      <string>File Selection</string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>19</y>
        <width>411</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QListView" name="file_list_view"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="add_files_btn">
           <property name="text">
            <string>Add Files</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="add_folder_btn">
           <property name="text">
            <string>Add Folder</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="remove_file_btn">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clear_files_btn">
           <property name="text">
            <string>Clear All</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>260</y>
       <width>431</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Export Settings</string>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>20</y>
        <width>411</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="export_path_edit"/>
       </item>
       <item>
        <widget class="QPushButton" name="browse_export_btn">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>431</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Migration Options</string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>19</y>
        <width>401</width>
        <height>100</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="preserve_conn">
         <property name="text">
          <string>Preserve Connections</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="preserve_formatting">
         <property name="text">
          <string>Preserve Formatting</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="create_log">
         <property name="text">
          <string>Create Log File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="include_data">
         <property name="text">
          <string>Include Data</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QPushButton" name="begin_migration_btn">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>480</y>
       <width>421</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Begin Migration</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>530</y>
       <width>411</width>
       <height>31</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>411</width>
      <height>601</height>
     </rect>
    </property>
    <property name="title">
     <string>Migration Monitor</string>
    </property>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>391</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Info</string>
     </property>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>371</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="3">
        <widget class="QLabel" name="files_processed">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Files Queued:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="files_queued">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Files Processed: </string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Process Time:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="process_time">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Failures:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="failures">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>391</width>
       <height>451</height>
      </rect>
     </property>
     <property name="title">
      <string>Migration Log</string>
     </property>
     <widget class="QTextBrowser" name="output_log">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>371</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="save_log_btn">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>420</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Log</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clear_log_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>420</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
